### ISSUE TITLE
OS X should be replaced with macOS

### ISSUE THREAD

**Issue by rojvv** [atom/atom.io#197]
### Description
According to Wikipedia:
> macOS (originally named "Mac OS X" until 2012 and then "OS X" until 2016) is the current Mac operating system that officially succeeded the classic Mac OS in 2001.

At "Everything you would expect" the cross-platform branch, OS X should be replaced with macOS.

### Steps to Reproduce

1. Visit [atom.io](https://atom.io) 
2. Scroll until you find "Everything you would expect"

![](https://i.ibb.co/RTG4wyN/screencapture-atom-io-2020-04-14-13-25-33-1.png)

**Reproduces how often:**

Every time when someone visits [atom.io](https://atom.io/) and scrolls down.

**Comment by DhruvDoshi** [comment]
Yes it seems like you are right !!
Just change it in readme file along with the source code and use it that way 
this is the advantage of Atom!!

**Comment by rojvv** [comment]
It is not fixed! What do you mean by "just change it in readme file along with the source code"?

**Comment by rojvv** [comment]
@lee-dohm 
@darangi 
@mtodd 

It is not fixed!

**Comment by lee-dohm** [comment]
@realRojSerbest Yes, we know it isn't fixed. We also intend to fix it at some point. That's why the issue is still open.

**Comment by realRojSerbest** [comment]
No one actually listens!
Why not close this **thread**?

**Comment by lee-dohm** [comment]
@realRojSerbest your comment was deleted as a violation of the [Atom Code of Conduct](https://github.com/atom/atom/blob/master/CODE_OF_CONDUCT.md) as it was insulting or derogatory. You may consider this an official warning.
